build.gradle

```diff
@@ -5,15 +5,19 @@ buildscript {
     ext.jna_version = '5.2.0'
     ext.android_gradle_plugin_version = '3.6.0'
 
-    // N.B. try to keep these two in sync
-    ext.android_components_version = '24.0.0'
-    ext.glean_version = '21.3.0'
```

Comment:
Note: the `glean_version` is now automatically published by the Glean Gradle plugin :-) We'll take care of keeping things in sync, no need for consuming products to do so!